content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,133 @@
+  - Pipelines
+  - Remote Storage
+---
+
+### [Q: Is it possible to plot metrics from multiple experiments together?](https://discord.com/channels/485586884165107732/563406153334128681/834387923482181653)
+
+You should be able to use experiment names in the `dvc plots` commands. You need
+to use the `diff` flag to compare multiple plots. Try
+`dvc plots diff exp-6ef18 exp-b17b4 exp-26e88`.
+
+Thanks to @PythonF from Discord for asking this question that led to this Gem!
+💎
+
+### [Q: Where are the configs for an experiment being pushed in Git when I run `dvc exp push`?](https://discord.com/channels/485586884165107732/563406153334128681/837773937390649364)
+
+It uses custom Git refs internally, similar to the way GitHub handles PRs. It’s
+a custom DVC Git ref pointing to a Git commit. Here's an example.
+
+```bash
+$ git show-ref exp-26220
+c42f48168830148b946f6a75d1bdbb25cda46f35 refs/exps/f1/37703af59ba1b80e77505a762335805d05d212/exp-26220
+```
+
+You can read more about how we handle our custom Git refs in
+[this blog post](https://dvc.org/blog/experiment-refs).
+
+Thanks to @Chandana for asking this question about experiments!
+
+### [Q: Is there a way to list all the experiments I have on my DVC remote that have not been committed to Git?](https://discord.com/channels/485586884165107732/563406153334128681/836705209039978538)
+
+Yes! The command `dvc exp list gitremote --all` lists all of the experiments for
+the default remote in the Git repo.
+
+Thanks again @Chandana for this gem!
+
+### [Q: Is CML compatible with Azure DevOps?](https://discord.com/channels/485586884165107732/728693131557732403/841664412221177926)
+
+Another great question from @Chandana!
+
+Right now, we support GitHub and GitLab.
+
+Azure DevOps and GCP (Google Cloud Platform) support are on the roadmap. Stay
+tuned for more updates!
+
+### [Q: I pushed a lot of files using `dvc push` to my DVC remote, but there are a few that couldn't be pushed at the time. If I run `dvc push` again, will it just upload the missing files?]()
+
+Thanks for the question @petek!
+
+Yes! You can just re-run `dvc push` and it will only upload the missing files.
+
+It might be a little slower than you would expect because DVC has to do some
+checks to make sure that the other files were uploaded successfully before, but
+as far as the actual data transfer goes, only the missing files will be
+uploaded.
+
+### [Q: After running `dvc repro`, I want to use `dvc add` on specific outputs of the pipeline, but I get this error: `output 'train' is already specified in stage: '../dvc.yaml`. Is there a way to manually add and pull data using the file versioning?](https://discord.com/channels/485586884165107732/485596304961962003/841688323663855616)
+
+Since the _dvc.lock_ file already has the data versioned, you could do a
+`dvc push`.
+
+If you want to keep the flexibility of adding and pulling data manually, there
+are a few things you can do. `dvc add` is specifically for tracking raw data. If
```

Comment:
I would start the answer with the `dvc add` sentence and drop everything before it. I think this will help frame the first paragraph as explaining the usage of `dvc add` and why it's not needed here.